chargebee_byte/requests/subscription_request.py

```diff
@@ -0,0 +1,36 @@
+from parameters import generate_sorting_parameters, generate_equals_parameters, generate_comparison_parameters, \
+    generate_collection_parameters, generate_date_parameters, generate_parameters
+
+
+class SubscriptionRequest(object):
+    path = '/subscriptions'
+
+    def __init__(self, parameters=None):
+        parameters = parameters or {}
+
+        self.allowed_parameters = self._generate_allowed_parameters()
```

Comment:
Minor performance nitpick, this will be re-invoked every time a request object is instantiated, but the allowed params per class will never change. Doesn't really matter though, we won't be instantiating 1000s of them anyway.

Reply:
I think it makes sense for now. We could probably delay the validation in some function call, but maybe we should wait until it becomes a problem? :)

Reply:
:+1: